data-prepper-plugins/obfuscate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/obfuscation/ObfuscationProcessorConfig.java

```diff
@@ -28,16 +30,25 @@ public class ObfuscationProcessorConfig {
     @JsonPropertyDescription("The source field to obfuscate.")
     @NotEmpty
     @NotNull
+    @ExampleValues({
+        @Example(value = "log", description = "The 'log' field in the input data will be obfuscated.")
+    })
     private String source;
 
     @JsonProperty("target")
     @JsonPropertyDescription("The new field in which to store the obfuscated value. " +
             "This leaves the original source field unchanged. " +
             "When no <code>target</code> is provided, the source field updates with the obfuscated value.")
+    @ExampleValues({
+        @Example(value = "newLog", description = "The 'newLog' field in the output data will have the input data obfuscated according to the other configurations that were set.")
+    })
     private String target;
 
     @JsonProperty("patterns")
     @JsonPropertyDescription("A list of regex patterns that allow you to obfuscate specific parts of a field. Only parts that match the regex pattern will obfuscate. When not provided, the processor obfuscates the whole field.")
+    @ExampleValues({
+        @Example(value = "[\"[A-Za-z0-9+_.-]+@([\\w-]+\\.)+[\\w-]{2,4}\"]", description = "This pattern represents an email address that will be obfuscated in the given field.")
```

Comment:
Let's remove the array and the outer double quotes. It should be clearer this way.

```suggestion
        @Example(value = "[A-Za-z0-9+_.-]+@([\\w-]+\\.)+[\\w-]{2,4}", description = "This pattern represents an email address that will be obfuscated in the given field.")
```